modules/shared/components/molecules/Misc/Misc.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import className from 'classnames';
+import type { FC, ReactElement } from 'react';
+import ErrorIcon from '../../icons/miscError.svg';
+import SuccessIcon from '../../icons/miscSuccess.svg';
+import styles from './Misc.module.css';
+import type { IMisc } from './types/IMisc';
+import * as EMisc from './types/EMisc';
+
+const Misc: FC<IMisc.IProps> = ({ type, msg, subMsg }): ReactElement => {
+  const classes: string = className({
+    [styles['success-box']]: type === EMisc.MiscType.Success,
+    [styles['error-box']]: type === EMisc.MiscType.Error,
+  });
+
+  return (
+    <div className={classes}>
+      {type === EMisc.MiscType.Success ? <SuccessIcon /> : null}
+      {type === EMisc.MiscType.Error ? <ErrorIcon /> : null}
+      <div className={styles['msg-box']}>
```

Comment:
i was wondering should we add default messages as these on the design [ Image couldn’t be uploaded! -- Congratulations ] and change them based on what is being passed with the type prop and make the msg/subMsg props optional for now.
i'd like to know your opinion.

Reply:
Thanks a lot that was a really good observation I have added them